judge/migrations/0088_compiler_message_ansi2html.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 2.1.12 on 2019-09-17 13:30
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import lxml.html as lh
+from lxml.html.clean import clean_html
+
+
+def strip_error_html(apps, schema_editor):
+    Submission = apps.get_model('judge', 'Submission')
+    for sub in Submission.objects.filter(error__isnull=False):
+        sub.error = clean_html(lh.fromstring(sub.error)).text_content() if sub.error else None
+        sub.save(update_fields=['error'])
```

Comment:
The better thing to do is to use [`Manager.bulk_update`](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#bulk-update) after we migrate to 2.2. And if we are doing that, don't buffer everything in memory, run `bulk_update` every 2k submissions or something.

Reply:
(Note for @Ninjaclasher: getting this PR in isn't blocked on 2.2.)